guides/common/modules/proc_deploying-the-ca-certificate-on-a-host-by-using-ansible-rex.adoc

```diff
@@ -0,0 +1,33 @@
+[id="deploying-the-ca-certificate-on-a-host-by-using-ansible-rex"]
+= Deploying the CA certificate on a host by using Ansible REX
+
+You can use remote execution (REX) with the Ansible provider to deploy the CA certificate.
```

Comment:
We're rather inconsistent about capitalizing remote execution. I see it's not listed under Capitalization in `CONTRIBUTING.md`. What's the guidance on this?

Reply:
I think we [agreed to decapitalize it](https://github.com/theforeman/foreman-documentation/issues/3140).